css/styles.css

```diff
@@ -78,14 +90,66 @@ button {
   padding: 10px 20px;
 }
 
+ul {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  padding: 5px 20px;
+}
+
+.entry-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+}
+
+.entry-header > a {
+  margin-right: 20px;
+  padding: 5px 22px;
+  border: 0 solid;
+  border-radius: 3px;
+  background-color: #562b81;
+  color: white;
+  text-decoration: none;
+}
+
+.entry-header > h2 {
+  letter-spacing: 1px;
+  font-size: 35px;
+  margin: 0;
+  padding-left: 15px;
+}
+
+.entry-photo > img {
+  width: 100%;
+  padding: 15px;
+  border: 0 solid;
+  border-radius: 30px;
+}
```

Comment:
Please ensure that you're using the sizing strategies in the `effective-css` guide: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css